src/main/java/io/appform/dropwizard/actors/common/RMQOperation.java

```diff
@@ -0,0 +1,8 @@
+package io.appform.dropwizard.actors.common;
+
+public enum RMQOperation {
+    CONSUME,
+    PUBLISH,
```

Comment:
Single publish operation should be required. No need for publish with different kind of attributes